browser.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env browser */
+import {createStringGenerator, createAsyncStringGenerator} from './core.js';
+
+const toHex = uInt8Array => uInt8Array.map(byte => byte.toString(16).padStart(2, '0')).join('');
+
+const decoder = new TextDecoder('utf8');
+const toBase64 = uInt8Array => btoa(decoder.decode(uInt8Array));
+
+// `crypto.getRandomValues` throws an error if too much entropy is requested at once. (https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues#exceptions)
+const maxEntropy = 65536;
+
+function getRandomValues(byteLength) {
+	const generatedBytes = [];
+
+	while (byteLength > 0) {
+		const bytesToGenerate = Math.min(byteLength, maxEntropy);
+		generatedBytes.push(crypto.getRandomValues(new Uint8Array({length: bytesToGenerate})));
+		byteLength -= bytesToGenerate;
+	}
+
+	const result = new Uint8Array(generatedBytes.reduce((sum, {byteLength}) => sum + byteLength, 0)); // eslint-disable-line unicorn/no-array-reduce
```

Comment:
`.reduce` allows us to write more readable and less code since the alternative is to do something [similar to `math-sum`](https://github.com/sindresorhus/math-sum/blob/aebaa07eb211a553fc8037b7657aa7473b88e17e/index.js#L4-L9).

Reply:
This is the one valid use for reduce, which the lint rule docs point out too 👍